pkgs/by-name/my/myanon/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, flex
+, bison
+}:
+
+stdenv.mkDerivation rec {
+  src = fetchFromGitHub {
+    owner = "ppomes";
+    repo = "myanon";
+    rev = "refs/tags/v0.5";
+    hash = "sha256-A4Qg8RvpkKgXoGTidshUyfX6FnhPR2ih1Nyne8Y87HA=";
+  };
+  pname = "myanon";
+  version = "0.5";
+
+  nativeBuildInputs = [ autoconf automake flex bison ];
+
+  configurePhase = ''
+    ./autogen.sh
+    ./configure --prefix=$out
+  '';
```

Comment:
Is this phase really needed?

Reply:
I guess, without this override I can't build this program. I suppose because it needs to run ./autogen.sh before configuring.